frontend/src/App.css

```diff
@@ -0,0 +1,26 @@
+:root {
```

Comment:
Lets create a separate style sheet for the features component instead of putting it in App.css. Can you create a styles folder and put the Features.css file in there.

Reply:
Will do!